model/group.go

```diff
@@ -69,23 +72,28 @@ type Group struct {
 	VaultRecovery string `json:"vaultRecovery,omitempty"`
 }
 
+// AddManager Add Account as Manager
 func (g *Group) AddManager(account *Account) {
-
 	g.addGroupAccount(account, GROUP_RIGHT_MANAGER)
-
 }
-func (g *Group) AddMember(account *Account) {
 
+// AddMember Add Account as Member
+func (g *Group) AddMember(account *Account) {
 	g.addGroupAccount(account, GROUP_RIGHT_MEMBER)
-
 }
 
+// AsPrimer Return Group with only Primer data
 func (g *Group) AsPrimer() *Group {
-
 	group := &Group{}
 	group.GroupPrimer = g.GroupPrimer
 	return group
+}
 
+// ToPrimer Convert to GroupPrimer
+func (g *Group) ToPrimer() *GroupPrimer {
+	groupPrimer := GroupPrimer{}
```

Comment:
volgens mij is deze assignment overbodig, de left-hand side variable wordt in de volgende regel overschreven